lib/metababel/bt2_matching_utils.rb

```diff
@@ -55,7 +55,7 @@ def match?(match_obj)
       end
 
       # We didn't match anything
-      if args_matched.empty? or args_matched.uniq.length != match_members.length
+      if args_matched.empty? || args_matched.uniq.length != match_members.length
```

Comment:
Only out of curiosity, why do we need this change?  `!=` has higher precedence than both `||` and `or`.

Reply:
Just for the sake of consistency. So we don't need to remember the subtle precedence difference.
 Either that or we use `()` everywhere.